Formula/angband.rb

```diff
@@ -20,21 +21,38 @@ class Angband < Formula
     sha256 x86_64_linux:   "3cfa3f60f7b88f2aa13cbbcae6b90e0aec128f5aef92e49c2860e61075c32af3"
   end
 
+  uses_from_macos "expect" => :test
+  uses_from_macos "ncurses"
+
   def install
-    ENV["NCURSES_CONFIG"] = "#{MacOS.sdk_path}/usr/bin/ncurses5.4-config"
-    system "./configure", "--prefix=#{prefix}",
-                          "--bindir=#{bin}",
-                          "--libdir=#{libexec}",
-                          "--enable-curses",
-                          "--disable-ncursestest",
-                          "--disable-sdltest",
-                          "--disable-x11",
-                          "--with-ncurses-prefix=#{MacOS.sdk_path}/usr"
+    ENV["NCURSES_CONFIG"] = "#{MacOS.sdk_path}/usr/bin/ncurses5.4-config" if OS.mac?
+    args = %W[
+      --prefix=#{prefix}
+      --bindir=#{bin}
+      --libdir=#{libexec}
+      --enable-curses
+      --disable-ncursestest
+      --disable-sdltest
+      --disable-x11
+    ]
+    args += "--with-ncurses-prefix=#{MacOS.sdk_path}/usr" if OS.mac?
```

Comment:
```suggestion
    args << "--with-ncurses-prefix=#{MacOS.sdk_path}/usr" if OS.mac?
```

Or can insert string inside an array.